BrainPortal/app/controllers/tool_configs_controller.rb

```diff
@@ -87,17 +87,25 @@ def report #:nodoc:
 
   def show #:nodoc:
     id     = params[:id]
-    config = ToolConfig.find(id)
+    @tool_config = ToolConfig.find(id)
+
+    # @config.group = Group.everyone if @config.group_id.blank?
+
+    @tool_config          = @tool_config
+    @tool_local_config    = @tool_config if   @tool_config.tool_id && @tool_config.bourreau_id # leaves nul otherwise
```

Comment:
Convenience? Keeping changes minimum to reduce error rate? tool_config indicates what config is shown actually (which could of all these 3 type: per tool, per bourreau, per combination). Tool_config_local attribute  describes a tool and bourreau specific config (if any), rest are global configs if any. I guess there might be a way to do everything with 3 only, e.g. with defining having tool_config = @tool_local_config || @tool_local_config || @bourreau_glob_config ;   show_table(tool_config) yet it is not job of controller to supply the data ready and let template concentrate on markup?

I also kept old edit page just in case (if edit accidentally popups somewhere), which is also defined on tool_config